pkgs/data/icons/hackneyed/default.nix

```diff
@@ -1,29 +1,41 @@
-{ lib, fetchzip, stdenvNoCC, fetchFromGitLab, xcursorgen, imagemagick6, inkscape }:
+{ lib, stdenvNoCC, fetchFromGitLab, imagemagick, inkscape, xcursorgen }:
 
 stdenvNoCC.mkDerivation rec {
   pname = "hackneyed";
-  version = "0.8.2";
+  version = "0.9.1";
 
   src = fetchFromGitLab {
     owner = "Enthymeme";
     repo = "hackneyed-x11-cursors";
     rev = version;
-    sha256 = "sha256-Wtrw/EzxCj4cAyfdBp0OJE4+c6FouW7+b6nFTLxdXNY=";
+    hash = "sha256-HKX3r5vsrMnzKhV3MkjeRUB93W64orH2BZTtXX6FLTQ=";
```

Comment:
```suggestion
    hash = "sha256-+7QtHgBuhJtQejiHeZ+QoedJo24LqSY51XRVLv9Ho2g=";
```

Reply:
thanks. I'm not sure why my machine keeps fetching a different hash.